src/test/interface/command_test.cpp

```diff
@@ -496,8 +496,9 @@ TEST(StanUiCommand, random_seed_fail_2) {
   model_path.push_back("test-models");
   model_path.push_back("transformed_data_rng_test");
 
-  int max = std::numeric_limits<int>::max();
-  unsigned int maxplus = max + 100;
+  long long int max = std::numeric_limits<unsigned int>::max();
+  long long int maxplus = max + 100;
+
```

Comment:
Add a test to make sure this runs with just max (equivalent of running the command in the PR).

I think if you check the error message isn't there with max and then is there with maxplus it's good enough:

```cpp
{
  std::string command
        = convert_model_path(model_path)
          + " sample num_samples=10 num_warmup=10 init=0 "
          + " random seed=" + std::to_string(max) + " "
          + " data file=src/test/test-models/transformed_data_rng_test.init.R"
          + " output refresh=0 file=test/output.csv";
    std::string cmd_output = run_command(command).output;
    run_command_output out = run_command(command);
    EXPECT_EQ(0, count_matches(expected_message, out.body));
}

{
  std::string command
        = convert_model_path(model_path)
          + " sample num_samples=10 num_warmup=10 init=0 "
          + " random seed=" + std::to_string(maxplus) + " "
          + " data file=src/test/test-models/transformed_data_rng_test.init.R"
          + " output refresh=0 file=test/output.csv";
    std::string cmd_output = run_command(command).output;
    run_command_output out = run_command(command);
    EXPECT_EQ(1, count_matches(expected_message, out.body));
}
```

Then seems good to me.

Reply:
Thanks, added!